src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -62,6 +62,22 @@ public static void executeCommand() {
         String[] readArgumentTokens = userCommandReader.getArgumentTokens();
         CommandList selectedCommand = CommandList.valueOf(readUserCommand);
 
+        if (MatchStat.getIsNewMatch()) {
```

Comment:
Does this condition indicate only when getIsNewMatch() is true, the user can input the HEAD, and TAIL commands.
Because during the game, we should prevent them use these commands.